File: CPP/Peripheral.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Peripheral : public Komponen{
	private:
		string tipe;
		string jenis_koneksi;
	
	public:
		Peripheral(){
			
		}
		
		Peripheral(string tipe, string jenis_koneksi, string merk, string nama) : 
		Komponen(merk, nama)
		{
			this->tipe = tipe;
			this->jenis_koneksi = jenis_koneksi;
		}
		
		void setTipe(string tipe){
			this->tipe = tipe;
		}
		
		void setJenisKoneksi(string jenis_koneksi){
			this->jenis_koneksi = jenis_koneksi;
		}
		
		string getTipe(){
			return this->tipe;
		}
		
		string getJenisKoneksi(){
			return this->jenis_koneksi;
		}
		
		~Peripheral(){
			
		}
};

File: Python/Program/Peripheral.py
from Komponen import Komponen

class Peripheral(Komponen):
    def __init__(self, tipe="", jenis_koneksi="", merk="", nama=""):
        super().__init__(merk, nama)
        self._tipe = tipe
        self._jenis_koneksi = jenis_koneksi

    def set_tipe(self, tipe):
        self._tipe = tipe

    def set_jenis_koneksi(self, jenis_koneksi):
        self._jenis_koneksi = jenis_koneksi

    def get_tipe(self):
        return self._tipe

    def get_jenis_koneksi(self):
        return self._jenis_koneksi